lib/data/models/activity_model.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:isar/isar.dart';
+import 'package:movna/domain/entities/sport.dart';
+
+part 'activity_model.g.dart';
+
+@Collection()
+class ActivityModel {
+  /// Use startTime as unique identifier.
+  Id get id => startTime.microsecondsSinceEpoch;
+
+  DateTime startTime = DateTime.fromMillisecondsSinceEpoch(0);
+
+  DateTime? stopTime;
+
+  // TODO : add field timezoneOffset or timezone to store localTime as well,
```

Comment:
yup excellent, same as in the adapter though, create an issue and link it (both TODOs will link the same issue)

EDIT: actually might as well add that field already